packages/core/src/indexing/migrations/1-create-model-table.ts

```diff
@@ -38,44 +38,48 @@ export type TableIndices = {
   indices: Array<TableIndex>
 }
 
+export function indexNameFromTableName(tableName: string): string {
```

Comment:
As discussed, needs to be more robust so we can stay within 64chars limit with variable field names.

Reply:
where did we land on this?  Is there a ticket somewhere capturing what remains to be done here?